### ISSUE TITLE
Feature: Auto-play on start

### ISSUE THREAD

**Issue by michael-hawker** [jenius-apps/ambie#130]
There should be a toggle in the settings to auto-play sounds when Ambie launches (as you already remember what was last selected).

**Comment by dpaulino** [comment]
I did some testing on this. While the golden path is nice, it can also lead to some scenarios with suboptimal UX. One time I opened the app and my OS volume was really high, the immediate playback was very jarring and unexpected because I forgot I had auto play turned on. Another design oddity is the inconsistency. How does one decide whether to auto play or not? Ambie remembers the last active items, but those items may not have been playing. Do we only auto play if they were playing at the time the window was closed? Or we auto play all the time? Auto playing all the time leads the potential jarring issues described above. And if we only selectively auto play, then the inconsistent experience when opening the app is weird since I did not know what UX the expect when launching ambie. 

In short, after experimentation, I haven't found an implementation that provides a consistent and delightful experience. So I'll close this ticket for now. Happy to reopen if others have ideas.
